nginx-proxy/cors.conf

```diff
@@ -1,4 +1,4 @@
-add_header 'Access-Control-Allow-Origin' '*';
+add_header 'Access-Control-Allow-Origin' '*' always;
```

Comment:
this was needed to handle CORS issue whenever the API returned an error:
https://stackoverflow.com/questions/28160982/how-to-make-nginx-cors-configuration-work-when-server-returns-error

Reply:
We can make look into making this part of the next version.